Source/ThirdParty/GltfPipeline/parseBinaryGltf.js

```diff
@@ -60,9 +60,9 @@ define([
 
             var contentString = getStringFromTypedArray(data, jsonStart, contentLength);
             gltf = JSON.parse(contentString);
-
-            // Clone just the binary chunk so the underlying buffer can be freed
-            var binaryData = new Uint8Array(data.subarray(binaryStart, length));
+            
+            // do not clone, TypedArray copies are a major performance bottleneck in IE11
+            var binaryData = data.subarray(binaryStart, length);
```

Comment:
Can you also make this fix in the `// Load binary glTF version 2` section?

Reply:
There are 2 other areas were the buffer is copied, and probably doesn't need to be. Could you also fix those areas?

https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/Batched3DModel3DTileContent.js#L345-L352
https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/Instanced3DModel3DTileContent.js#L301-L308

Each can just be `var gltfView = new Uint8Array(arrayBuffer, byteOffset, gltfByteLength);` - but may require some testing.